loader/icd_dispatch.c

```diff
@@ -847,8 +847,8 @@ clEnqueueReadBufferRect(
     cl_command_queue command_queue,
     cl_mem buffer,
     cl_bool blocking_read,
-    const size_t * buffer_origin,
-    const size_t * host_origin, 
+    const size_t * buffer_offset,
+    const size_t * host_offset, 
```

Comment:
Are you suggesting to open a PR on the headers to change the naming of this parameter (and the others you commented on)?

Reply:
I just had another thought: I put `readability-inconsistent-declaration-parameter-name` into the `WarningsAsErrors` list.

Do we want to support slightly mismatching Header/ICD versions? Because putting it there would break that.